packages/fluent-theme/src/fluent/styles/Dialog.styles.ts

```diff
@@ -17,7 +17,7 @@ export const DialogContentStyles = (props: IDialogContentStyleProps): Partial<ID
     title: {
       fontSize: FontSizes.size20,
       fontWeight: FontWeights.semibold,
-      padding: '16px 24px 24px 24px',
+      padding: '16px 46px 24px 24px',
```

Comment:
I wonder if this would be better done as a flexbox (Stack is now in place so we could just fix the dialog to use that) so you don't have to have this kind of adjustment.

Reply:
@kenotron I don't think a flexbox solution will work here as the `X` button is positioned absolutely with a specific space between the top and right edge of the Dialog container. If we go with a flex solution this will need to be first gone through a designer review to make sure the intended redlines are not broken.

Reply:
On second thought it could still work with a flexbox but would that be just adding more code to achieve the same thing as regular padding did))

Reply:
Also, as a side note the issue is manifesting only in fluent theme due to decreased padding comparing with the current default styles.